src/components/item_bids/ItemBids.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from 'react';
+import BidCard from "../bid_card/BidCard";
+
+const ItemBids = ({ item_uuid, apiHandler, handleError }) => {
+  const [bids, setBids] = useState([]);
+
+  useEffect(() => {
+    const fetchBids = async () => {
+      const { data, errors } = await apiHandler.get(`/api/items/${item_uuid}/bids`);
+      data ? setBids(data) : handleError(errors);
+    }
+    fetchBids();
+  }, [apiHandler, item_uuid]);
+
+  const renderBids = (bidCollection) => {
+    return bidCollection.map(bid => {
+      return <BidCard white={true} key={bid.uuid} timestamp={bid.timestamp} bid_price={bid.bid_price} userName={"Anonymous"} />
+    })
+  }
+
+  if (bids.length > 0) {
+    return (
+      <>
+        <h2 className='header--secondary'>Bids</h2>
+        {renderBids(bids)}
```

Comment:
Why did you make a `renderBids` function when the only thing you're rendering in addition to the bid cards is the `h2`? Is it just because you wanted to hide the map logic? I don't think it's worth it (but that's my personal preference)

Reply:
I wanted to hide the map logic, but I'll update to just put it right in render